.github/actions/setup-action/action.yml

```diff
@@ -57,3 +57,8 @@ runs:
         github_repository: ${{ github.repository }}     
         github_current_run_id: ${{ github.run_id }}
         pull_request_url: ${{ github.event.issue.pull_request.url }}
+    ### Used for Flink jobs that call docker run and mount gcloud folder
```

Comment:
The failure is due to setup-action.yml gets overwritten (by checkout action) during the workflow run, and parser running into inconsistency state. Likely a github issue.

----

If this is for certain flink job only, then we do not need it add to setup-action which every job run?

Reply:
We decided to put it there as to hide the implementation from the people creating new workflows. 
Its not necessarily just for Flink but other jobs could use gcloud. 

Its not compute intensive so really not an issue to keep it there

Reply:
> The failure is due to setup-action.yml gets overwritten (by checkout action) during the workflow run, and parser running into inconsistency state. Likely a github issue.

Should we ignore it or? Its unclear to me why it happens so im not sure how to apporach the fix

Reply:
Removed the flink reference.

Ill try to look from time to time how the action behaves and is this something that is happening often.